apps/modernization-ui/src/apps/page-builder/pages/AddNewPage/AddNewPageFields.tsx

```diff
@@ -140,8 +141,16 @@ export const AddNewPageFields = (props: AddNewPageFieldProps) => {
             <Controller
                 control={form.control}
                 name="dataMartName"
-                render={({ field: { onChange, value } }) => (
-                    <Input label="Data mart name" type="text" onChange={onChange} defaultValue={value} />
+                rules={dataMartNameRule}
```

Comment:
Suggestion: Moving the `maxLengthRule` here would increase reusability of the `validation/entry/dataMartNameRule`

Reply:
Presumed, like the syntax rules, maxLength rule would be applied to all instances of dataMartName?
This follows the pattern of other fields being validated here.